src/sentry/workflow_engine/models/workflow.py

```diff
@@ -38,8 +38,20 @@ class Workflow(DefaultFieldsModel, OwnerModel, JSONConfigBase):
 
     @property
     def config_schema(self) -> dict[str, Any]:
-        # TODO: fill in
-        return {}
+        return {
+            "$schema": "https://json-schema.org/draft/2020-12/schema",
+            "$id": "https://example.com/product.schema.json",
```

Comment:
should this be `workflow` or something along those lines rather than example.com? (or should we just remove this line? seems like it's meant to allow us to identify this specific schema/subschema)

Reply:
oh yes let me remove